.gitignore

```diff
@@ -1,5 +1,9 @@
 # Created by https://www.gitignore.io/api/node,vim,macos,linux,windows
 
+# data directory, used by aws-mocks:
+*/data
```

Comment:
This may ignore our test image. I would probably do just `data` or maybe `./data`.